WordPress/Classes/ViewRelated/Me/SettingsCommon.swift

```diff
@@ -4,24 +4,44 @@ protocol SettingsController: ImmuTableController {}
 
 // MARK: - Actions
 extension SettingsController {
-    func editText(changeType: (AccountSettingsChangeWithString), hint: String? = nil, service: AccountSettingsService) -> ImmuTableRowControllerGenerator {
+    func editText(changeType: AccountSettingsChangeWithString,
+                  hint: String? = nil,
+                  isEmail: Bool = false,
+                  isPassword: Bool = false,
+                  isPresented: Bool = false,
+                  service: AccountSettingsService) -> ImmuTableRowControllerGenerator
```

Comment:
I'm not too happy about all these toggles.

Instead of `isPresented`, maybe `ImmuTablePresenter.present(_:)` should handle adding the wrapping navigation controller.

If we're not editing passwords so far, I'd drop the parameter.

As for `isEmail`, perhaps `editEmailAddress` shouldn't call `editText`, but instead create a function that calls `controllerForEditableText` and set its `isPassword` property.